costmap_2d/include/costmap_2d/inflation_layer.h

```diff
@@ -194,4 +195,4 @@ class InflationLayer : public Layer
 
 }  // namespace costmap_2d
 
-#endif  // COSTMAP_2D_INFLATION_LAYER_H_
\ No newline at end of file
+#endif  // COSTMAP_2D_INFLATION_LAYER_H_
```

Comment:
In the future, please try not to include these nonfunctional changes in functional PRs.